distsql/distsql.go

```diff
@@ -78,9 +78,7 @@ func Select(ctx context.Context, sctx sessionctx.Context, kvReq *kv.Request, fie
 		hook.(func(*kv.Request))(kvReq)
 	}
 
-	if !sctx.GetSessionVars().EnableStreaming {
-		kvReq.Streaming = false
-	}
+	kvReq.Streaming = false
```

Comment:
It looks like that `Streaming` field for `Request` can be removed as well? Anyway even if so, it doesn't have to be done in this PR.